### PR TITLE
[CLOUD-3003] - untag EAP tests moved to jboss-eap-modules

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-3003
Signed-off-by: Ken Wills <kwills@redhat.com>

See: https://github.com/jboss-container-images/jboss-eap-modules/pull/37

Thanks for submitting your Pull Request!

Please make sure your PR meets following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull request does not include fixes for other issues than the main ticket
- [ ] Attached commits represent unit of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@redhat.com>` - use `git commit -s`

### CODE REVIEW

**Comment by luck3y** [comment]
@bstansberry @rcernich FYI

File: tests/features/eap/6.4/eap.feature [hunk 1]
```diff
@@ -1,26 +1,25 @@
-@jboss-eap-6/eap64-openshift
+
 Feature: Openshift EAP basic tests
 
-  @ci
-  Scenario: Check that the jboss-eap-6/eap64-openshift image contains 6 layers
+  Scenario: Check that the  image contains 6 layers
```
Comment (maschmid) [inline tests/features/eap/6.4/eap.feature]: This doesn't seem correct

File: tests/features/eap/6.4/eap.feature [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
     Given image is built
      Then image should contain 6 layers
 
   # https://issues.jboss.org/browse/CLOUD-180
   Scenario: Check if image version and release is printed on boot
     When container is ready
-    Then container log should contain Running jboss-eap-6/eap64-openshift image, version
+    Then container log should contain Running  image, version
```
Comment (maschmid) [inline tests/features/eap/6.4/eap.feature]: This doesn't seem correct

File: tests/features/eap/6.4/eap.feature [hunk 3]
```diff
@@ -1,26 +1,25 @@
...
 
   Scenario: Check that the labels are correctly set
     Given image is built
      Then the image should contain label com.redhat.component with value jboss-eap-6-eap64-openshift-container
-      And the image should contain label name with value jboss-eap-6/eap64-openshift
+      And the image should contain label name with value
```
Comment (maschmid) [inline tests/features/eap/6.4/eap.feature]: no value

File: tests/features/eap/6.4/eap.feature (hunk 1 above)
Comment (luck3y) [reply tests/features/eap/6.4/eap.feature]: Yep, thanks for spotting these, will update.

File: tests/features/common.feature [hunk 4]
```diff
@@ -1,7 +1,7 @@
 
 Feature: Openshift common tests
 
-  @jboss-eap-6/eap64-openshift @jboss-eap-7 @jboss-decisionserver-6 @jboss-processserver-6 @jboss-webserver-3/webserver30-tomcat7-openshift @jboss-webserver-3/webserver31-tomcat7-openshift @jboss-webserver-3/webserver30-tomcat8-openshift @jboss-webserver-3/webserver31-tomcat8-openshift @jboss-amq-6 @jboss-datagrid-6 @jboss-datagrid-7
+    @jboss-decisionserver-6 @jboss-processserver-6 @jboss-webserver-3/webserver30-tomcat7-openshift @jboss-webserver-3/webserver31-tomcat7-openshift @jboss-webserver-3/webserver30-tomcat8-openshift @jboss-webserver-3/webserver31-tomcat8-openshift @jboss-amq-6 @jboss-datagrid-6 @jboss-datagrid-7
```
Comment (maschmid) [inline tests/features/common.feature]: not sure I like the extra whitespace at the beginning of the lines...

File: tests/features/eap/7/eap_7_resourceadapters.feature [hunk 5]
```diff
@@ -30,15 +30,15 @@ Feature: EAP 7 Openshift resource adapters
     Then XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value activemq-rar.rar on XPath //*[local-name()='resource-adapter']/@id
      And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value activemq-rar.rar on XPath //*[local-name()='resource-adapter']/*[local-name()='archive']
      And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value org.apache.activemq.ra.ActiveMQManagedConnectionFactory on XPath //*[local-name()='resource-adapter']/*[local-name()='connection-definitions']/*[local-name()='connection-definition']/@class-name
-     And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value java:/ConnectionFactory on XPath //*[local-name()='resource-adapter']/*[local-name()='connection-definitions']/*[local-name()='connection-definition']/@jndi-name
+     And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value java:/ConnectionFactory on XPath //*[local-name()='resource-adapter']/*[local-name()='connection-definitions']/*[local-name()='connection-definition']/jndi-name
```
Comment (maschmid) [inline tests/features/eap/7/eap_7_resourceadapters.feature]: sure we want to do this?

File: tests/features/common.feature (hunk 4 above)
Comment (luck3y) [reply tests/features/common.feature]: Yeah, same, I thought I'd just fixed all of them... Will double check.

File: tests/features/eap/7/eap_7_resourceadapters.feature (hunk 5 above)
Comment (luck3y) [reply tests/features/eap/7/eap_7_resourceadapters.feature]: Good, catch, thanks.

Comment (luck3y) [reply tests/features/eap/7/eap_7_resourceadapters.feature]: Thx, fixed. I checked the other instances of this too and they're fine, so odd this one got caught up. It was probably an accident from initially starting these substitutions with perl -i -e 's/..../.../g' then realising all those @'s in perl were just going to be a pain and switching to sed ;)

**Pull request merged**